ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11-periodics.yaml

```diff
@@ -18707,7 +18707,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build03
-  cron: 17 3 16 2 *
+  cron: 0 0 1 9 *
```

Comment:
@kabirbhartiRH Since you changed the job name, you have to run `make update` again to update these files.